release/release_logs/2.5.0/benchmarks/many_actors.json

```diff
@@ -0,0 +1,32 @@
+{
+    "_dashboard_memory_usage_mb": 476.291072,
+    "_dashboard_test_success": true,
+    "_peak_memory": 5.93,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n253\t2.03GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n1806\t0.86GiB\tpython distributed/test_many_actors.py\n347\t0.34GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dashboa\n451\t0.09GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n61\t0.09GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n1409\t0.07GiB\tray::JobSupervisor\n54\t0.07GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-lab --allow-root --ip=127.0.0.1 --no-\n2039\t0.07GiB\tray::DashboardTester.run\n1947\t0.06GiB\tray::MemoryMonitorActor.run\n346\t0.04GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=10.0.36.81:6379 --host=0.0.0.0 --",
+    "actors_per_second": 789.1952586839337,
+    "num_actors": 10000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "actors_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 789.1952586839337
+        },
+        {
+            "perf_metric_name": "dashboard_p50_latency_ms",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 23.104
+        },
+        {
+            "perf_metric_name": "dashboard_p95_latency_ms",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 1231.637
+        },
+        {
+            "perf_metric_name": "dashboard_p99_latency_ms",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 4195.452
```

Comment:
Yeah I think this is fine - we haven't been tracking this, I am not sure how reliable this metric is actually (variance wise). Should this even be in the release log?

Reply:
Yeah this doesn't need to be tracked. I think it has a high variance, and I am tracking the perf just as a reference purpose (not to actually track the performance). I will do sth by next release to avoid confusion..